_posts/2022/2022-12-05-find-list-debug-active-record-callbacks-in-the-console.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Fimd out what Callbacks are defined on an Active Record model in the console"
+description: "Sometimes it's hars to know"
+layout: article
+category: ruby
+image:
+  base: "2022/find-list-debug-active-record-callbacks-in-the-console"
+  alt: "Three old payphones"
+  credit: "Pavan Trikutam"
+  source: "https://unsplash.com/photos/71CjSSB83Wo"
+---
+
+I was recently asked by my [First #RubyFriend mentee](https://firstrubyfriend.org) how to list the callbacks present on an Active Record model. I didn't know.
```

Comment:
I think 'mentee' shouldn't be included in the link! (A super minor point!)

Reply:
```suggestion
I was recently asked by my [First #RubyFriend](https://firstrubyfriend.org) mentee how to list the callbacks present on an Active Record model. I didn't know.
```